.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
I would prefer that the files not be added under .idea.

Reply:
I am not getting it?

Reply:
As part of this pull request, you have added several files under .idea folder. You may want to delete them from your git branch and then [modify the .gitignore file to not add them](https://stackoverflow.com/questions/17049416/what-is-the-idea-folder).

Reply:
I've made the change to the main branch to add `.idea` to the `.gitignore` file